docs/ldml/tr35-dates.md

```diff
@@ -1706,12 +1706,16 @@ Note: A generic location format is constructed by a part of time zone ID represe
 * "UTC-3" (short)
 * "Гринуич+03:30" (long)
 
-Otherwise (when the offset from GMT is zero, referring to GMT itself) the style specified by the `<gmtZeroFormat>` element is used:
```

Comment:
Actually I don't agree with the assumption that a zero GMT offset refers to GMT itself. Azores Summer Time is GMT+0, but that doesn't make it GMT, it's still Azores Summer Time, just with a GMT offset of zero. I would actually prefer if this format never used the `gmtZeroFormat` and we wouldn't need to introduce a new format for this behaviour.

Reply:
We didn't have time in the last design meeting to discuss this (and I assume it is targeted to v48),

Reply:
No this is 47